package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "@eth-optimism/superchain-registry",
+  "version": "0.0.1-alpha.1",
+  "description": "[Optimism] Superchain Registry",
+  "author": "Optimism",
+  "private": false,
+  "keywords": [
+    "superchain",
+    "optimism",
+    "op-stack",
+    "ethereum"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ethereum-optimism/superchain-registry.git",
+  },
+  "files": [
+    "superchain/",
+    "assets/"
+  ]
```

Can you add a `.npmignore` file to the root of the repo, and ignore all Go and Markdown files, with `*.go` and `*.md` globs?

---

Alternatively you can add it here to 

```suggestion
    "!**/*.go",
    "!**/*.md"
  ]
```